app/views/passengers/index.haml

```diff
@@ -21,7 +21,7 @@
         %tr
           %td.will_expire_soon Will Expire Within 1 Week
           %td.expired_within_grace_period Expired Within Grace Period
-          %td.has_been_overridden Expiration Overridden by Admin
+          %td.overridden Expiration Overridden by Admin
```

Comment:
Will there ever be a time when the expiration is overridden by a different user? In other words, do we need to specify that the expiration is overridden by admin?

Reply:
It's not for specificity, just for clarity, I imagine. Personally I think we might do another `s/Admin/STSPV` to make it even clearer who we're talking about.

But yes, only admins can override expiration dates.

Reply:
That sounds like a good idea -- I like STSPV over Admin.